packages/vats/test/test-priceAuthorityRegistry.js

```diff
@@ -3,8 +3,8 @@ import { makeScalarBigMapStore } from '@agoric/vat-data';
 import { AmountMath, makeIssuerKit } from '@agoric/ertp';
 import { buildManualTimer } from '@agoric/swingset-vat/tools/manual-timer.js';
 import { E } from '@endo/eventual-send';
-import { providePriceAuthorityRegistry } from '../../../tools/priceAuthorityRegistry.js';
-import { makeFakePriceAuthority } from '../../../tools/fakePriceAuthority.js';
+import { providePriceAuthorityRegistry } from '../src/priceAuthorityRegistry.js';
+import { makeFakePriceAuthority } from '../../zoe/tools/fakePriceAuthority.js';
```

Comment:
any reason to not import from `@agoric/zoe`? *lint-primary* is showing cross-package import as an error.